### ISSUE TITLE
minify makes shl operation (<<) bigger

### ISSUE THREAD

**Issue by neophob** [babel/minify#748]
Hi

thanks for babel-minify! I use it for a js1k compo and discovered that the code `array.fill(0xff<<24 | (time%16 ? 0 : 0x38));` is minified to `g.fill(-16777216|(n%16?0:56))`.

I use minify without any additional parameter, only `node_modules/babel-minify/bin/minify.js index.js`

#### Expected Output

```js
`g.fill((0xff<<24)|(n%16?0:56))`
```

**Comment by j-f1** [comment]
```js
g.fill(-16777216|(n%16?0:56)) // actual
g.fill((0xff<<24)|(n%16?0:56)) // expected
```

Looks like the current minification is actually one character shorter.

**Comment by neophob** [comment]
hi @jf-1 - sorry my bad the corrected expected output is `g.fill(-16777216|(n%16?0:56))`.

**Comment by jogibear9988** [comment]
Should it only minify, or also look for performant code?

Maybe -16777216 is performanter as 0xff<<24

**Comment by neophob** [comment]
Good point yes - maybe that should be an option to go for max compression or max speed?
